Resources/Migrations/whiteMigration.yml

```
@@ -0,0 +1,2 @@
+# 2024-11-10
+VendingMachinePride: VendingMachineCola
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)